documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -83,17 +109,137 @@ Other options for the `hibernate.search.elasticsearch.index_management_strategy`
 |CREATE_DELETE|Similarly to 'CREATE' but will also delete the index at shutdown. Commonly used for tests.
 |===============
 
-==== Updating the indexes
+===== Elasticsearch configuration
+
+There is no specific configuration required on the Elasticsearch side.
+
+However there are a few features that would benefit from a few changes:
+
+* analyzers should be defined in the Elasticsearch configuration, see <<elasticsearch-mapping-analyzer>>
+* if you want to retrieve the distance in a geolocation query, install and enable the `lang-groovy` plugin,
+  see <<elasticsearch-query-spatial>>
+* if you want to be able to delete by query, install the `delete-by-query` plugin
+
+=== Mapping and indexing
+
+Like in Lucene embedded mode, indexes are transparently updated when you create or update
+entities mapped to Hibernate Search.
+Simply use familiar annotations from <<search-mapping>>.
+
+The name of the index will be the lowercased name provided to `@Indexed` (non qualified class name by default).
+Hibernate Search will map the fully qualified class name to the Elasticsearch type.
+
+==== [[elasticsearch-mapping-analyzer]] Analyzers
+
+CAUTION: Analyzers are treated differently than in Lucene embedded mode.
 
-As with Lucene in embedded mode, the indexes are updated automatically when you create or update
-entities which are mapped to Hibernate Search using the same annotations already familiar from <<search-mapping>>.
+Analyzers should be defined (by name) in your Elasticsearch configuration
+instead of using `@AnalyzerDef`, we are looking into smoothing that experience.
```

Comment:
if ES adds dynamic or index creation time analyzer definition, we could. Do you know if that's in their plans?